rust-runtime/aws-smithy-types/src/date_time/mod.rs

```diff
@@ -404,7 +404,7 @@ mod test {
 
     #[derive(Debug)]
     struct EpochMillisTestCase {
-        rfc3339: &'static str,
+        _rfc3339: &'static str,
```

Comment:
These fields were unused and it kept warning me when running tests so I underscored them

Reply:
Yeah, they were intended to make the test json file more human readable, but weren't actually needed by the tests.